ai-server/app/router/team_recommendation_router.py

```
@@ -0,0 +1,226 @@
+        # 2. 매칭 점수 순으로 정렬
+        candidates.sort(key=lambda x: x.matchScore, reverse=True)
+        
+        # 3. 요청된 팀 크기만큼 반환
+        result = candidates[:request.teamSize]
+        
+        logger.info(f"✅ 팀원 추천 완료 - 추천된 팀원 수: {len(result)}")
+        return result
+        
+    except Exception as e:
+        logger.error(f"❌ 팀원 추천 실패: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"팀원 추천 중 오류가 발생했습니다: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)